chg: creator e-mail in the event details, fixes #1252

cristianbell opened the pull request:
#### What does it do?

displays the creator e-mail in the event details for org admins, fixes #1252
#### Questions
- [ ] Does it require a DB change?
- [ ] Are you using it in production?
- [ ] Does it require a change in the API (PyMISP for example)?
#### Release Type:
- [ ] Major
- [x] Minor
- [ ] Patch

cristianbell committed:
chg: creator e-mail in the event details, fixes #1252

cvandeplas reviewed (changes requested):
The current View/view.ctp already contains the feature to show the email adress to the siteadmin. For a normal user (or org_admin) this info is not shown.

So the view() function in this Controller already fetches that information somewhere. This same code should be extended to also allow visibility for the org_admin for events of his org. 
Hint: $this->Event->fetchEvent($this->Auth->User(), $conditions); 

Not relevant as this code should be removed:
- The isOwnedByOrg function has no value, except doing an additional DB lookup. In the $results variable you already have the event info. Use that var to check if the event org is the same as the user org. (or org_id)
- The getUserMinimalInfo function is also not needed, as it's covered by an earlier query.

cvandeplas commented:
Ok, sorry about the review, ignore what's inside... I tried a new feature of Github, and failed using / removing / updating it.
The remarks I have are: 

The `isOwnedByOrg` function has no value, except doing an additional DB lookup. In the `$results`variable you already have the event info. Use that var to check if the event org is the same as the logged in user org. (or org_id)

For the `getUserMinimalInfo($id)` function there are 2 options: 
1. Remove the function and search for the data immediately `$this->User->field('email', array('id' , $results[0]['Event']['user_id']))` 
2. Rename the function to something that says what it does: for example: `getUserEmailById($id)` that returns a string with the email address.    (and there is no need to query/return the id in the fields as you already have it, and it's not used in the existing code)

cristianbell committed:
chg: creator e-mail in the event details, fixes #1252
- removed use of isOwnedByOrg()
- removed getUserMinimalInfo(), replaced with $this->User->field('email'..)

cristianbell commented:
- removed use of isOwnedByOrg()
- removed getUserMinimalInfo(), replaced with $this->User->field('email'..)
